main.go

```diff
@@ -593,15 +644,18 @@ func writeBadData(bw *spanner.BatchWriter, conv *internal.Conv, banner, name str
 	fmt.Fprintf(out, "See file '%s' for details of bad rows\n", name)
 }
 
-func getDatabaseName(now time.Time) (string, error) {
-	return generateName(fmt.Sprintf("pg_dump_%s", now.Format("2006-01-02")))
+func getDatabaseName(SQL string, now time.Time) (string, error) {
+	switch SQL {
+	case MYSQL:
+		return generateName(fmt.Sprintf("mysql_%s", now.Format("2006-01-02")))
+	case POSTGRES:
+		return generateName(fmt.Sprintf("pg_dump_%s", now.Format("2006-01-02")))
```

Comment:
I have not touched the already existing functionality in which `pg_dump_` is the database name, but added case for `mysql_`
Please suggest if I should change it to pg or postgres instead of pg_dump ?
We can also keep it according to the driverName.

Reply:
I would prefer to use `driverName`, which is more specific.

Reply:
https://github.com/cloudspannerecosystem/harbourbridge/blob/fa3d015562a86e51157a983f900b07a1fb5ca62d/main.go#L641-L643
Done